capture-sdk/sdk/src/main/java/net/gini/android/capture/tracking/OnboardingScreenEvent.java

```diff
@@ -9,15 +9,15 @@
 /**
  * Events triggered on the onboarding screen.
  *
- * <p> All events trigger on both Screen and Component APIs.
+ * <p> All events trigger on Screen APIs.
  */
 public enum OnboardingScreenEvent {
     /**
-     * Triggers when the first onboarding page is shown.(<b>Screen API + Component API</b>)
+     * Triggers when the first onboarding page is shown.(<b>Screen API</b>)
```

Comment:
```suggestion
     * Triggers when the first onboarding page is shown.
```

Reply:
@zladzeyka you call this small changes haha :D I missed here a lot. No problem I'm checking right away.